dashboards/scylladb/scylladb-prometheus.json

```diff
@@ -52,25 +51,31 @@
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "scylla_cache_bytes_used{${Cluster},${Location},${Namespace}}"
+                  "prometheusQuery": "scylla_cache_bytes_used{${Cluster},${Location},${Namespace}}",
+                  "unitOverride": ""
                 }
               }
             ],
+            "thresholds": [],
             "timeshiftDuration": "0s",
             "yAxis": {
+              "label": "",
               "scale": "LINEAR"
             }
           }
         },
         "width": 12,
         "xPos": 0,
-        "yPos": 28
+        "yPos": 7
```

Comment:
Does this mean we have to retake the screenshot preview of the dashboard in the same folder?